scripts/utils.js

```diff
@@ -0,0 +1,42 @@
+/* @flow */
+
+// eslint-disable-next-line import/no-nodejs-modules
+import fs from "fs";
+
+type Package = {|
+  name: string,
+  version: string,
+|};
+
+type NodeOps = {|
+  web: {|
+    staticNamespace: string,
+  |},
+|};
+
+export function getPackage(): Package {
```

Comment:
Are we able to just `import` package.json directly in the buildLogos.js file? Wondering if using `fs` for this is overkill.